shell/browser/net/electron_url_loader_factory.cc

```diff
@@ -189,6 +194,24 @@ void ElectronURLLoaderFactory::Clone(
   receivers_.Add(this, std::move(receiver));
 }
 
+std::string ComputeMethodForRedirect(const std::string& method,
+                                     int http_status_code) {
+  // For 303 redirects, all request methods except HEAD are converted to GET,
+  // as per the latest httpbis draft.  The draft also allows POST requests to
+  // be converted to GETs when following 301/302 redirects, for historical
+  // reasons. Most major browsers do this and so shall we.  Both RFC 2616 and
+  // the httpbis draft say to prompt the user to confirm the generation of new
+  // requests, other than GET and HEAD requests, but IE omits these prompts and
+  // so shall we.
+  // See: https://tools.ietf.org/html/rfc7231#section-6.4
```

Comment:
I believe this should be implemented upstream, not by us.

See https://source.chromium.org/chromium/chromium/src/+/master:services/network/cors/cors_url_loader.cc;l=359-367;drc=e3aec06168fddef3f2a085fb1f6282b2b9920480